packages/timeline-state-resolver/src/devices/vizMSE.ts

```diff
@@ -36,16 +36,15 @@ import {
 	VRundown,
 	InternalElement,
 	ExternalElement,
-	VElement,
-	VProfile,
-	VizEngine
+	VElement
 } from 'v-connection'
 
 import { DoOnTime, SendMode } from '../doOnTime'
 
 import * as crypto from 'crypto'
 import * as net from 'net'
 import { ExpectedPlayoutItem } from '../expectedPlayoutItems'
+import * as request from 'request'
```

Comment:
It would be nice to not use request if possible as it is EOL, but I know we are using it elsewhere in this repo so I'm not going to block this on that